pulp_docker/app/views.py

```diff
@@ -222,6 +246,6 @@ def get(self, request, path):
         tags = {'name': path, 'tags': []}
         for c in distribution.publication.repository_version.content:
             c = c.cast()
-            if isinstance(c, Tag):
+            if isinstance(c, ManifestTag) or isinstance(c, ManifestListTag):
```

Comment:
You need to add some logic here to make sure you are not listing a tag with the same name twice.

Reply:
Good call. Switched to a set.